.travis.yml

```diff
@@ -18,28 +18,84 @@ language: java
 
 matrix:
   include:
-    - name: "Ubuntu 14.04 / Java 8 / OpenSSL 1.0.x"
+
+    - name: "x64 / Ubuntu 14.04 / Java 8 / OpenSSL 1.0.x"
```

Comment:
Should this be the same as ppc64le and aarch64 build and be Ubuntu 16.04?

Reply:
I left it as 14.04 as that is what it was before. An arguable advantage of using an older Ubuntu is that it will have an older GCC so you can ensure wider compatibility.

Unfortunately 14.04 isn't available for ppc64le and aarch64 on Travis, which is why I used 16.04 instead.

Reply:
I don't fully agree, but this is still a minor nit.  Probably fine to keep as is.

Reply:
Okay no worries. Could you explain? I am happy to change it if it makes sense...